__tests__/schema/thread/edit-comment.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * This file is part of Serlo.org API
+ *
+ * Copyright (c) 2020-2022 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2020-2022 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/api.serlo.org for the canonical source repository
+ */
+import { gql } from 'apollo-server'
+
+import { article, comment1, user } from '../../../__fixtures__'
```

Comment:
```suggestion
import { article, comment, user } from '../../../__fixtures__'
```
`comment1` gives the impression you used more than one comment in the tests. 
Any reason why you imported `comment1` and not simply `comment`?
If `comment1` really is what you want, please do 
`import { artical, comment1 as comment, user} ...`

Of course the other parts in code have to be adjusted accordingly.